lib/cloud_controller/config.rb

```diff
@@ -22,15 +22,7 @@ class InvalidConfigPath < StandardError
     class << self
       def load_from_file(file_name, context: :api, secrets_hash: {})
         config = VCAP::CloudController::YAMLConfig.safe_load_file(file_name)
```

Comment:
You can replace this `safe_load_file` line with `read_file`.

So `load_from_file` does `read_file` + `load_from_hash`.